monticore-grammar/src/main/java/de/monticore/types3/SymTypeRelations.java

```diff
@@ -253,13 +253,24 @@ public static boolean isByte(SymTypeExpression type) {
    * This returns whether the type is _exactly_ String.
    * In most cases, you want to check whether the type
    * is either compatible to, or a subtype of String.
+   * You may want to use
+   * {@link #isStringOrSubType(SymTypeExpression)} instead.
    */
   public static boolean isString(SymTypeExpression type) {
     return getDelegate()._isString(type);
   }
 
   protected abstract boolean _isString(SymTypeExpression type);
 
+  /**
+   * whether it is a String (boxed or unboxed) or a subtype (e.g., a RegEx)
```

Comment:
```suggestion
   * @param type the SymTypeExpression to check
   * @return whether it is a String (boxed or unboxed) or a subtype (e.g., a RegEx)
```

(to make the javadoc happy)